examples/segmentationBrush/index.html

```diff
@@ -110,63 +114,95 @@ <h5>Controls:</h5>
       imageIds: imageIds
   };
 
-  cornerstone.enable(element);  
+  cornerstone.enable(element);
   cornerstone.loadImage(imageIds[0]).then(function(image) {
-    cornerstone.addLayer(element, image);
-    cornerstone.updateImage(element);
-    var viewport = cornerstone.getViewport(element);
-    viewport.pixelReplication = true;
-    cornerstone.setViewport(element, viewport);
+    // TEMP -- This line demonstrates the issue experienced in the OHIF viewer. Ideally displayImage would be called first and then the tool deals with adding an additional layer.
+    cornerstone.displayImage(element, image);
```

Comment:
// TEMP... I guess we should get rid of this..

Reply:
Fixed.